apps/research-service/src/validations/validators/feature-bounds.validator.ts

```diff
@@ -0,0 +1,133 @@
+import { GeometryValidator, ValidationResult } from "./validator.interface";
+import { Geometry, Polygon, MultiPolygon, Point } from "geojson";
+
+interface FeatureBoundsValidationResult extends ValidationResult {
+  extraInfo: {
+    invalidCoordinates: Array<{
+      longitude: number;
+      latitude: number;
+      reason: string;
+    }>;
+  } | null;
+}
+
+export class FeatureBoundsValidator implements GeometryValidator {
+  async validateGeometry(
+    geometry: Geometry,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars -- Required by GeometryValidator interface
+    _properties?: Record<string, unknown>
```

Comment:
You can simply leave this param off, and it should still be a valid implementation:
```
async validateGeometry(geometry: Geometry): Promise<FeatureBoundsValidationResult> {
```